lib/exnumerator/string_list_enum.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Exnumerator.StringListEnum do
+  def cast(values, term) do
+    if term in values do
+      {:ok, term}
+    else
+      :error
+    end
+  end
+
+  def load(values, term) do
+    if term in values do
+      {:ok, term}
+    else
+      :error
+    end
+  end
+
+  def dump(values, term) do
+    if term in values do
+      {:ok, term}
+    else
+      :error
+    end
+  end
+end
```

Comment:
I believe you will benefit from delegating that behaviour to a single common function.

Reply:
@KamilLelonek I just focused on porting the code and then thing about DRY.

I started super DRY and I had to remove it 😄